tests/testthat/test-bainancovabayesian.R

```diff
@@ -1,51 +1,69 @@
 context("Bain ANCOVA")
```

Comment:
Did you create new tests based on the rewrite?
The reason I ask is because quite bit seems to have shifted around and it's definitely good to test the results of the old analysis against those of the rewrite.

Reply:
I updated the existing tests by running jaspTools with `makeTests = TRUE` but I did not change the contents of the tests.

Reply:
Okay, but did you run the old tests also against the rewritten code? Because newly created tests will obviously pass, but you want the old tests to also still pass (that's the real test)